.github/pull_request_template.md

```diff
@@ -52,3 +52,11 @@ If you answered '_yes_' to any of the questions above, conduct a detailed Review
     - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
 - Have you ensured logging does not contain sensitive data?
 - Have you received any additional approvals needed for this change?
+
+## Guiding questions for Author and Reviewer:
+
+- Do these changes meet/improve code clarity?
+	- Do these changes introduce any magic?
```

Comment:
```suggestion
	- Do these changes avoid introducing any magic?
```

All the other questions are phrased "Do these changes `MAKE THE WORLD BETTER`", except for this one.   I propose reversing this, to parallel the others.